.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "test/spec/test-cases"]
 	path = test/spec/test-cases
-	url = https://github.com/ChainSafe/eth2.0-spec-tests.git
+	url = https://github.com/ethereum/eth2.0-spec-tests.git
```

Comment:
@mpetrunic @wemeetagain AFAIK this should remain chainsafe, perhaps for the purposes of this branch `dev/0.7.1` we can change this line to be a branch with the updated tests?

Reply:
yup, I told him to reference ethereum repo for simplicity until it's ready to merge

Reply:
So should tests be passing ?

Reply:
I am updating spec tests. There are some errors. Trying to remove them.

Reply:
Data structures have been changed in 0.7.1. but I only updated it for stateTransition. So, spec tests are failing. I think if we have a complete merged version  then all the spec tests will pass.